vermouth/processors/name_moltype.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Assign moltype names to molecules.
```

Comment:
Too succinct

Reply:
What else should be there that is not already in the docstring of the processor itself? Or that should be repeated?

Reply:
Some repetition might not hurt

Reply:
I can copy paste half the docstring from just bellow. It does increase the odds of having a mismatch between the code and the doc, though.

Reply:
"Provides the processor ..., which can be conveniently used to ..."